docs/fundamentals/toc.yml

```
@@ -942,6 +942,10 @@ items:
           - name: Overview
             displayName: "language rules"
             href: code-analysis/style-rules/language-rules.md
+          - name: 'this.' and 'Me.' qualifiers (IDE0003 and IDE0009)
```

I am in two minds on whether this should be:
- 'this.' and 'Me.' qualifiers (IDE0003 and IDE0009) OR
- IDE0003 and IDE0009: 'this.' and 'Me.' qualifiers

I feel the former is better as users mostly case about the actual code style option group/name, rule ID being not that critical for style rules. @mikadumont @gewarren thoughts?

---

Hmm, I could go either way too, but I think I lean more towards putting the ID first.

```suggestion
          - name: IDE0003 and IDE0009 ('this' and 'Me' qualifiers)
```